users/konstantin/config.h

```diff
@@ -31,7 +31,16 @@
 #define MOUSEKEY_WHEEL_TIME_TO_MAX 50
 
 // RGB lighting
-#undef  RGBLIGHT_ANIMATIONS
+#undef RGBLIGHT_EFFECT_BREATHING
```

Comment:
A few of these, like this one, now have `#undef RGBLIGHT_EFFECT_xxxx`, immediately followed by `#define RGBLIGHT_EFFECT_xxxx`.  I guessing that's because a script was used to transform `#undef RGBLIGHT_ANIMATIONS`, but it does make things a little messy.

I'm not sure whether it's worth fixing though...

Reply:
Wasn't a script 😅 
But figured it was easier to undef all and the redefine, to ensure that the intended behavior was maintained.

Reply:
Wow... Human script!
No worries - the objection was `__attribute__((weak))`...